INSTALL.md

```diff
@@ -0,0 +1,73 @@
+# Build weshnet
+
+These are instructions to build weshnet.
+
+<!-- markdownlint-disable MD034 -->
+
+## Prerequisites
+
+* Required: asdf
+* Required: asdf Go and Go Lint plugins
+* Required on macOS: Command Line Developer Tools
+
+Following are the steps to install each prerequisite (if it's needed for your
+build target).
+
+### macOS 11, macOS 12 and macOS 13
+
+To install the Command Line Developer Tools, in a terminal enter:
+
+    xcode-select --install
+
+After the Developer Tools are installed, we need to make sure it is updated. In
+System Preferences, click Software Update and update it if needed.
+
+To install asdf using brew, follow instructions at https://asdf-vm.com . In short,
+first install brew following the instructions at https://brew.sh . Then, in
+a terminal enter:
+
+    brew install asdf
+    echo -e "\n. $(brew --prefix asdf)/libexec/asdf.sh" >> ${ZDOTDIR:-~}/.zshrc
+
+Start a new terminal to get the changes to .zshrc .
+
+To install the Go and Go Lint plugins, enter:
+
+    asdf plugin add golang
+    asdf plugin add golangci-lint
+
+### Ubuntu 18.04, 20.04 and 22.04
+
+To install asdf, follow instructions at https://asdf-vm.com . In short, in
+a terminal enter:
+
+    sudo apt install curl git
+    git clone https://github.com/asdf-vm/asdf.git ~/.asdf
+    echo '. "$HOME/.asdf/asdf.sh"' >> ~/.bashrc
+
+Start a new terminal to get the changes to .bashrc .
+
+To install the Go and Go Lint plugins, enter:
+
+    asdf plugin add golang
+    asdf plugin add golangci-lint
```

Comment:
I made a PR (merged) where I wrote a rule (`asdf.install_tools`) in the Makefile to install the dependencies: https://github.com/berty/weshnet/pull/17
I let you decide if you prefer to keep the doc like this or change it to use the Makefile rule. Personally, I prefer that the doc refers to the Makefile rule, because it will be easier to update only the Makefile if we need to update dependencies and don't risk forgetting to update the doc.

Reply:
I changed to use `make asdf.install_tools` .